src/icons/text.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+
+const iconSize = Object.freeze({
+  lg: 'w-3.5 h-3.5',
+  md: 'w-3 h-3',
+  sm: 'w-2.5, h-2.5',
+})
+
+const Text = ({ className, size, ...props }) => {
+  return (
+    <svg className={`${iconSize[size]} ${className}`} viewBox="0 0 16 20" fill="currentColor" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
@Hamaad102 please look at the other files for icons & arrange props & sections similarly.